Source/Core/VideoCommon/NativeVertexFormat.h

```diff
@@ -63,6 +63,8 @@ struct PortableVertexDeclaration
   std::array<AttributeFormat, 8> texcoords;
   AttributeFormat posmtx;
 
+  PortableVertexDeclaration() { memset(this, 0, sizeof(*this)); }
```

Comment:
You may want to also add the static assert after the struct declaration:

```cpp
static_assert(std::is_trivially_copyable_v<PortableVertexDeclaration>);
```

to ensure that case never becomes false, since using memset like that is undefined behavior if it is.